docs/topics/vs-code-extension-interface.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/vsc-extension-guide/master.adoc
 
+:_content-type: REFERENCE
 [id="vs-code-extension-interface_{context}"]
```

Comment:
@sbeskin-redhat @stoobie Maybe I'm too close to this since I wrote it, but I think it's a concept module. It comes before procedure modules that refer to the locations described in this module.

Reply:
@RichardHoch  The module describes elements of a UI. Why is it a concept?

Reply:
@sbeskin-redhat A concept module explains the product or its components, usually before a procedure in which the user does something with those components.  A reference module gives additional or technical information, usually appearing after a procedure. For example, a list of keyboard shortcuts would be a reference-- it's extra, advanced information.
@stoobie -- what do you think? Is a description of a UI "concept" or "reference"? Maybe I'm drawing an incorrect distinction.

Reply:
@sbeskin-redhat @RichardHoch I think this is a CONCEPT. I'm going to read it once and not refer to it again. The information is not structured in a way that encourages scanning.